homeassistant/components/waze_travel_time/sensor.py

```diff
@@ -63,11 +64,13 @@ async def async_setup_entry(
     defaults = {
         CONF_REALTIME: DEFAULT_REALTIME,
         CONF_VEHICLE_TYPE: DEFAULT_VEHICLE_TYPE,
-        CONF_UNITS: hass.config.units.name,
+        CONF_UNITS: UNITS_METRIC,
```

Comment:
Checking/Storing the `name` of the unit system will stop working if we rename or add a new unit system.
It is more future-proof to check the `length_unit` of the unit system, even if the component still uses "imperial" terminology in the config flow.

Reply:
Adjusted to use `hass.config.units == ***`